Bottom: e235274bb4c5790989dbe96d0182c67444a066b3
Top:    c588e8ceae2b1e4b9184ab89c740bac7531d1ca7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:07:12 +0000

src/print.c: Fix fab drawing width text annotation.

The text hard-coded that the outline is 10mil, in fact it is not,
the current #define sets it at 8 mil. Report the actual value used.


---

diff --git a/src/print.c b/src/print.c
index b5fec1c..cd19d31 100644
--- a/src/print.c
+++ b/src/print.c
@@ -319,9 +319,9 @@ PrintFab (void)
 	       "Maximum Dimensions: %d mils wide, %d mils high",
 	       PCB->MaxWidth / 100, PCB->MaxHeight / 100);
       text_at (PCB->MaxWidth / 2, PCB->MaxHeight + 2000, 1,
-	       "Board outline is the centerline of this 10 mil"
+	       "Board outline is the centerline of this %d mil"
 	       " rectangle - 0,0 to %d,%d mils",
-	       PCB->MaxWidth / 100, PCB->MaxHeight / 100);
+	       FAB_LINE_W / 100, PCB->MaxWidth / 100, PCB->MaxHeight / 100);
     }
   else
     {
